client/asset/btc/electrum_test.go

```diff
@@ -0,0 +1,165 @@
+// This code is available on the terms of the project LICENSE.md file,
+// also available online at https://blueoakcouncil.org/license/1.0.0.
+
+//go:build electrumlive
+
+package btc
+
+import (
+	"bytes"
+	"context"
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"testing"
+	"time"
+
+	"decred.org/dcrdex/client/asset"
+	dexltc "decred.org/dcrdex/dex/networks/ltc"
+	"github.com/btcsuite/btcd/btcutil/psbt"
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/btcsuite/btcd/wire"
+)
+
+const walletPass = "walletpass"
+
+func TestPSBT(t *testing.T) {
+	// an unsigned swap txn funded by a single p2wpkh
+	txRaw, _ := hex.DecodeString("010000000118f36e05994f8e69dace16f4a3d4c9ff8db6f58bf48d498b1c0d6daf63c6f8690100000000ffffffff02408e2c0000000000220020e0133024bb27f510f6959467df91ed5daa791f598ba1c09ef1f1ac4540bfdda16e7710000000000016001460a8dbedd538501e2ad9e9f5a4d8a11e35a6c4e700000000")
+	msgTx := wire.NewMsgTx(wire.TxVersion)
+	err := msgTx.Deserialize(bytes.NewReader(txRaw))
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	packet, err := psbt.NewFromUnsignedTx(msgTx)
+	if err != nil {
+		t.Fatal(err)
+	}
+	enc, err := packet.B64Encode()
+	if err != nil {
+		t.Fatal(err)
+	}
+	fmt.Println(enc)
+}
+
+func TestElectrumExchangeWallet(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	tipChanged := make(chan struct{}, 1)
+	walletCfg := &asset.WalletConfig{
+		Type: walletTypeElectrum,
+		Settings: map[string]string{
+			"rpcuser":     "user",
+			"rpcpassword": "pass",
+			"rpcbind":     "127.0.0.1:5678",
+		},
+		TipChange: func(error) {
+			select {
+			case tipChanged <- struct{}{}:
+				t.Log("tip change, that's enough testing")
+				time.Sleep(300 * time.Millisecond)
+				cancel()
+			default:
+			}
+		},
+		PeersChange: func(num uint32) {
+			t.Log("peer count: ", num)
+		},
+	}
+	cfg := &BTCCloneCFG{
+		WalletCFG:           walletCfg,
+		Symbol:              "ltc",
+		Logger:              tLogger,
+		ChainParams:         dexltc.TestNet4Params,
```

Comment:
This confused me a little. Should this test go in **client/assets/ltc**? Either way is fine.